charts/pega/config/deploy/context.xml.tmpl

```diff
@@ -21,20 +21,20 @@
     minEvictableIdleTimeMillis="60000"
     />
 
-  {{ if and .Env.DB_RW_USERNAME .Env.JDBC_RW_URL }}
+  {{ if or .Env.SET_RW .Env.JDBC_RW_URL }}
```

Comment:
new env var: `SET_RW` which if set and no `JDBC_RW_URL` set will take the url for main PegaRULES datasource. The intended behavior of this datasource *was* to always be same as main PegaRULES just with longer timeouts. By still including JDBC_RW_URL we allow for backwards compatibility